.github/workflows/claude.yml

```
@@ -19,36 +19,32 @@ jobs:
             actions: read
 
-          claude_args: |
-            --allowedTools "Bash(uv:*),Bash(uvx:*),Bash(rg:*),Bash(grep:*),Bash(sqlite3:*),Bash(alembic:*)"
+          # Optional: Give a custom prompt to Claude. If this is not specified, Claude will perform the instructions specified in the comment that tagged it.
+          # prompt: 'Update the pull request description to include a summary of changes.'
+
+          # Optional: Add claude_args to customize behavior and configuration
+          # See https://github.com/anthropics/claude-code-action/blob/main/docs/usage.md
+          # or https://code.claude.com/docs/en/cli-reference for available options
+          # claude_args: '--allowed-tools Bash(gh pr:*)'
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)